modules/currency-l1/src/main/scala/org/tessellation/currency/l1/http/DataApplicationRoutes.scala

```diff
@@ -57,14 +53,19 @@ final case class DataApplicationRoutes[F[_]: Async: Hasher: SecurityProvider: L1
                   case Invalid(e) => InternalServerError(InvalidDataUpdate(e.toString).toApplicationError)
                   case Valid(_) =>
                     dataUpdatesQueue.offer(hashed.signed) >>
-                      Ok(("hash" ->> hashed.hash.value) :: HNil)
+                      Ok(("hash" -> hashed.hash.value).asJson)
```

Comment:
I changed this to use Circe directly as the use of shapeless was causing an issue at some point. I don't recall exactly the issue but it was something like "unable to find Encoder[<complexShapelessThing>]". Using Circe reduces the overall number of deps in this file and replicates the previous behavior but if there is a non-obvious reason this should not be changed, I can revert back.

Reply:
That's okay :+1:

Reply:
@scasplte2 oh, I remember now. Are you able to use the new syntax with multiple fields (multiple types)? I think that was why we used shapeless.